gradle-plugin/src/test/java/org/projectnessie/nessierunner/gradle/TestQuarkusApp.java

```diff
@@ -34,8 +34,8 @@
 import org.junit.jupiter.api.io.TempDir;
 
 /**
- * Test for the {@link QuarkusAppPlugin}, which basically simulates what the {@code build.gradle} in
- * Apache Iceberg does.
+ * Test for the {@link NessieRunnerPlugin}, which basically simulates what the {@code build.gradle}
+ * in Apache Iceberg does.
  */
 class TestQuarkusApp {
```

Comment:
worth renaming as well?
maybe find/grep codebase for `QuarkusApp`

Reply:
renamed as well